packages/dialtone-vue2/components/combobox/combobox.stories.js

```diff
@@ -3,14 +3,14 @@ import { createRenderConfig } from '@/common/storybook_utils';
 import DtCombobox from './combobox.vue';
 
 import DtComboboxDefaultTemplate from './combobox_default.story.vue';
-import { LABEL_SIZES } from './combobox_constants';
+import { COMBOBOX_LABEL_SIZES } from './combobox_constants';
```

Comment:
Renamed to match our specifications about prefixing the constants with the component name.